### PR TITLE
Reply system

### PR DESCRIPTION
**Pull request by Tripe2000**
Working reply system, you can reply to a question. When clicking the form button, it will save the reply to the database if the user is logged in and there is content in the reply body. QUESTION ID IS SET TO 1, needs to be fixed when the question page is well implemented

### CODE REVIEW

**Review by tongweiz — COMMENTED** [review]
I saw it, but it may need more review.

File: soen341-UB3-QA/app/Http/Controllers/ReplyController.php [hunk 1]
```diff
@@ -35,9 +34,24 @@ public function create()
...
+			$reply->question_id = $qid;
+			$reply->user_id = Auth::id();
+			$reply->content = request('body');
+			$reply->likectr = 0;
+			$reply->dislikectr = 0;
+			$reply->status = 0;
+		
+			$reply->save();
+		}
+		return redirect("question/$qid");
```
Comment (irinafrazao) [inline soen341-UB3-QA/app/Http/Controllers/ReplyController.php:53]: redirect to /details. The question view is not formatted. We should use /details view for that page

File: soen341-UB3-QA/routes/web.php [hunk 2]
```diff
@@ -9,23 +8,16 @@
 | routes are loaded by the RouteServiceProvider within a group which
 | contains the "web" middleware group. Now create something great!
 */
-
 Route::get('/', function () {
     return view('welcome');
 });
```
Comment (irinafrazao) [inline soen341-UB3-QA/routes/web.php:13]: use /details routing

**Review by irinafrazao — CHANGES_REQUESTED** [review]
You should use /details view instead of /question. /question doesnt have the formatting. /details is the one we will use for your page. Also, did you test this? I couldnt get to your page from button and I cant test the commit submit button. Check your routing. (more comments in file on github)

**Review by irinafrazao — APPROVED** [review]

**Pull request closed without merging**